docs/index.md

```diff
@@ -34,7 +34,7 @@ The `splunk-add-on-ucc-framework`:
 * generates OpenAPI description documents (`appserver/static/openapi.json` file) (for more information, see [here](openapi.md)).
 * generates `.conf` files (more information, see [here](dot_conf_files.md)).
 * installs Python requirements (`lib` folder).
-* generate metadata files (`metadata` folder).
+* generate [metadata](./metadata.md) files (`metadata` folder).
```

Comment:
If the `metadata.md` file points to description for `output/Splunk_addon/metadata` directory, I think there is wrong content in the `.md` file. This file's content is for globalConfig's meta data and not the `default.meta` that is generated by ucc-gen.

Reply:
yea, i mixed those 2, i will remove these reference